concent_api/core/migrations/0003_auto_20180109_1436.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-01-09 14:36
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def migrate_golem_messages_types_from_strings_to_ints(apps, schema_editor):
+
+    Message = apps.get_model('core', 'Message')
+    for message in Message.objects.all():
+        if message.type == 'ForceReportComputedTask':
+            message.tmp_type = 4001
+        if message.type == 'AckReportComputedTask':
+            message.tmp_type = 4002
+        if message.type == 'RejectReportComputedTask':
+            message.tmp_type = 4003
+        if message.type == 'VerdictReportComputedTask':
+            message.tmp_type = 4004
```

Comment:
You should use `elif` if the `if` does not terminate the loop/function. If you don't Python always has to check all the conditions for every item instead of stopping immediately after finding one that matches.

Reply:
If statements changed to elif. All fixUps squashed.